src/pages/Donors/Donors.test.tsx

```diff
@@ -13,6 +37,68 @@ describe(
     title: "Donors",
     privatePage: true,
     withBackButton: false,
-    element: <Donors />,
+    element: (
+      <ApolloProvider client={createGoodChatClient()}>
+        <Donors />
+      </ApolloProvider>
+    ),
   })
 );
+
+test("should show a list of conversations", async () => {
+  const { container } = render(
+    <ApolloProvider client={createGoodChatClient()}>
+      <Donors />
+    </ApolloProvider>
+  );
+
+  await wait(() =>
+    expect(container.querySelector("ion-item")).toBeInTheDocument()
+  );
+  expect(container.querySelectorAll("ion-item")).toHaveLength(2);
```

Comment:
You want to check the text content of the items. `ion-item` can be something in the page which isn't the conversation